library/ccm.c

```diff
@@ -85,6 +107,18 @@ int mbedtls_ccm_setkey(mbedtls_ccm_context *ctx,
                                      MBEDTLS_ENCRYPT)) != 0) {
         return ret;
     }
+#else
+    if (keybits != 128 && keybits != 192 && keybits != 256) {
```

Comment:
Shouldn't cipher_light take care of this validation? It doesn't depend on the mode. I don't think it matters to preserve the exact error code returned for that case.

Reply:
> Shouldn't `cipher_light` take care of this validation?

It does, because the low-level `setkey()` functions do. This was just a quick way to make the tests pass by returning the same error code as before. We might do it differently in a production PR.